README.md

```diff
@@ -37,7 +38,8 @@ const rules = {
   username: 'required|unique:users,username',
   email: 'required|email',
   age: 'required|numeric|min:18',
-  gender: 'in:male,female'
+  gender: 'in:male,female',
+  phoneNumber: 'regex:[0-9]{3} ?-?[0-9]{3} ?-?[0-9]{3}'
```

Comment:
Test this with `,`. It's used to separate parameters. If you get more that 1 parameter then you need to join parameteres with `,`.